content/blog/vlan-versus-vxlan.md

```diff
@@ -0,0 +1,93 @@
+
+### VLAN Header in Ethernet Frame
+
+![VLAN Header](/img/picture1.png "VLAN Header")
+
+* The ethernet frame has 4 bytes VLAN tag field to identify the VLAN-ID that the packet originated from. The format of VLAN-tagged frames is defined in IEEE 802.1Q standard.
+* The VLAN TAG has below fields:
+
+  * TPID – By Default VLAN Tagged packets has 0x8100.
+  * Priority – To indicate the 802.1p priority of the frame
+  * CFI – By default it is 0. 0 indicates MAC addresses are in standard format. 1 indicated non-standard format.
+  * VLAN-ID - 12 Bit value to identify the VLAN.
+
+### VXLAN Header in Ethernet Frame
+
+![VXLAN Header](/img/picture2.png "VXLAN Header")
+
+* The entire ethernet frame including DMAC, SMAC and IP-header or IPv6 header and UDP header is encapsulated with four headers:
+  outer ethernet header, outer IP-header, outer UDP Header and VXLAN Header.
+* VXLAN Header has the following fields:
+
+
+  * Flags -  I bit would be set to 1 for a valid VXLAN Network ID (VNI).  The other reserved 7 bits would be set to zero.
+  * VXLAN-ID: This is the 24 bit value used to designate the VXLAN overlay network.
+
+### VLAN and VXLAN Configuration:
+
+To understand VXLAN further, let us go through the configurations of VLAN and VXLAN. 
+Each VNI would be mapped with the VLAN it is carrying.
+
+![VLAN, VXLAN Configuration](/img/picture3.png "VLAN, VXLAN Configuration")
+
+### FAQs
+
+**1. Will VXLAN replace VLAN standard?**
+
+VLAN standard is specified in IEEE 802.1Q standard by IEEE.
+VXLAN standard is specified in RFC 7438 by IETF.
+VXLAN encapsulates the VLAN tagged packet with four outer headers to transmit the packet in IP based network. It does not replace the VLAN standard.
+
+**2. Does VXLAN always have outer VLAN header?**
+
+The outer VLAN header in VXLAN headers is optional. It is based on the deployment scenario.
+
+**3. Does VXLAN only solve the scalability issue?** 
+
+Along with scalability issue, VXLAN is used for having IP based traffic in core network. 
+As VXLAN is used for network segmentation which solves two major issues:
+Allows multiple VMs to share a single physical network without seeing each other’s traffic for the reuse of IP addresses.
+The VNI identifies the scope of the inner MAC frame originated by the individual VM.  Thus, it could have overlapping MAC addresses across VXLAN segments but never have traffic "cross over" since the traffic is isolated using the VNI.
+
+**4. What’s the difference between QinQ and VXLAN?**
+
+QinQ and VXLAN are protocols designed for achieving virtual segmentation of physical network. QinQ adds an outer VLAN header on top of the original ethernet frame. This also solves scalability issues.
+Whereas VXLAN is tunneling mechanism which is primarily layer3 over layer2 ethernet frame. VXLAN also adds outer UDP header.
+
+To summarize, VLAN and VXLAN are used together to solve scalability issues and to overcome the limitations of VLAN.
```

Comment:
I recommend adding a call to action statement at the end here. Something perhaps that says "To learn more about VLANs and VXLANs, please check out this documentation here (and then provide a link)." It could even be an external link. Or, if you will be doing a Munch & Learn or Meetup with Didier, you could refer them to that for more details.

Reply:
Done